pydotorg/settings/base.py

```diff
@@ -349,6 +351,10 @@
 
 GLOBAL_SURROGATE_KEY = 'pydotorg-app'
 
+### pydotorg.settings.cabotage.add_surrogate_keys_to_static
+
```

Comment:
do we clear caches on deploy for pydotorg-app? Just wondering if we could just do it all in one instead of special casing static files

Reply:
No, and that's intentional. Purging `pydotorg-app` is more of a "break glass" situation since that key covers hundreds of pages that will need to be re-populated.